tour-tip/get-tip-model.js

```diff
@@ -0,0 +1,24 @@
+import tipsConfig from './config';
+
+function deepCopy(obj) {
+	return JSON.parse(JSON.stringify(obj));
+}
+
+export function getById(id) {
+	const tipFound = tipsConfig.find(tip => tip.id === id);
+	if (!tipFound) {
```

Comment:
you could also have if tipFound return deepCopy(tipFound) rather than checking for a negative result first, because a function that doesn't return anything will return undefined